feature-libs/quote/components/summary/actions/quote-summary-actions.component.ts

```diff
@@ -71,6 +137,177 @@ export class QuoteSummaryActionsComponent implements OnInit, OnDestroy {
     });
   }
 
+  protected isDesktop() {
```

Comment:
Is it either desktop or mobile? Then we could have _one_ method instead of 2. Does that make sense?

Reply:
Agree!